media-video/ffmpeg/ffmpeg-4.4.1-r4.ebuild

```diff
@@ -563,4 +564,6 @@ multilib_src_install() {
 multilib_src_install_all() {
 	dodoc Changelog README.md CREDITS doc/*.txt doc/APIchanges
 	[ -f "RELEASE_NOTES" ] && dodoc "RELEASE_NOTES"
```

Comment:
(I know this isn't your ebuild, but I want you to know we prefer double brackets, `[[ ]]`